### PR TITLE
normalizer with unit std

### PR DESCRIPTION
**Pull request by hnyu**
Add an option for using normalizers with zero mean only (unit variance). Additionally support target updater for single nn.Parameter

### CODE REVIEW

File: alf/algorithms/data_transformer.py [hunk 1]
```diff
@@ -600,11 +600,21 @@ def __init__(self,
         self._update_mode = update_mode
 
     def _transform(self, timestep_or_exp):
+        norm = self._normalizer
         if ((self._update_mode == "replay" and common.is_replay())
                 or (self._update_mode == "rollout" and common.is_rollout())):
-            self._normalizer.update(timestep_or_exp.reward)
+            norm.update(timestep_or_exp.reward)
+
+            with alf.summary.scope("RewardNormalizer"):
```
Comment (emailweixu) [inline alf/algorithms/data_transformer.py]: This part should be a member function of Normalizer as it accesses private members.

Comment (hnyu) [reply alf/algorithms/data_transformer.py:608]: Deleted. Just realized that the normalizer already has a summary function inside

Comment (hnyu) [reply alf/algorithms/data_transformer.py]: > This part should be a member function of Normalizer as it accesses private members.

Deleted. Just realized that the normalizer already has a summary function inside

Comment (emailweixu) [reply alf/algorithms/data_transformer.py]: Make sure you are aware that normalizer won't summarize if debug_summaries is not True

Comment (hnyu) [reply alf/algorithms/data_transformer.py]: > Make sure you are aware that normalizer won't summarize if debug_summaries is not True

Yeah I've passed a flag in the gin file

**Review by emailweixu — APPROVED** [review]

**Pull request merged**